### PR TITLE
Furniture price additions

### PR DESCRIPTION
**Pull request by Aeltorian**
**_I affirm:_**
- [x] I understand that if I do not agree to the following points by completing the checkboxes my PR will be ignored.
- [x] I understand I should leave resolving conversations to the LandSandBoat team so that reviewers won't miss what was said.
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/LandSandBoat/server/blob/base/CODE_OF_CONDUCT.md).
- [x] I have _**tested my code and the things my code has changed**_ since the last commit in the PR and will test after any later commits.

## What does this pull request do?

Adds proper base prices to quite a few of furniture that were previously set to 0. Verified base prices from several sources.

## Steps to test these changes

Sell the items to the store. They should not show up as 0 selling price.

### CODE REVIEW

File: sql/item_mods.sql [hunk 1]
```diff
@@ -67293,6 +67293,13 @@ INSERT INTO `item_mods` VALUES (27594,288,4); -- DOUBLE_ATTACK: 4
...
 INSERT INTO `item_mods` VALUES (27595,28,12); -- MATT: 12
 
+-- Emico Mantle
+INSERT INTO `item_mods` VALUES (27597,1,12);     -- DEF: 12
+INSERT INTO `item_mods` VALUES (27597,2,20);     -- HP: 20
+INSERT INTO `item_mods` VALUES (27597,10,3);     -- VIT: 3
+INSERT INTO `item_mods` VALUES (27597,18,10);    -- EARTH_RES: 10
+INSERT INTO `item_mods` VALUES (27597,27,4);     -- ENMITY: 4
+
 -- Dew Silk Cape
```
Comment (Xaver-DaRed) [inline sql/item_mods.sql:67303]: You did not create a different branch for the sale values? This is already in your other PR.

**Comment by Xaver-DaRed** [comment]
> Verified base prices from several sources.

And were did you get this values from exactly?

File: sql/item_mods.sql (hunk 1 above)
Comment (Aeltorian) [reply sql/item_mods.sql:67303]: I thought I did. Carver was walking me through and we did make a branch of it. 

edit: wait, why is this edit here too? Eh, i'm still learning this structuring of this. My apologies. I may have likely not a proper pull before starting the new branch therefore catching the other edit. Again,  my apologies. Rough start. Lol

**Comment by Aeltorian** [comment]
> > Verified base prices from several sources.
> 
> And were did you get this values from exactly?

I reviewed several websites; BG-wiki if they had any, then I branched out to other various sites like ffxiah, ffxiopedia, and various pictures on google image search of sell numbers. I tried jp-wiki but couldn't read/translate fast enough. Initially, this was to fix the sell base number of Girandola due to the report but I just saw a bunch of 0 sell base numbers and thought about due diligence. So, I just did the ones around that area of the sql. If this isn't doesn't comply, I can withdraw this no problem.

**Comment by TeoTwawki** [comment]
> > > Verified base prices from several sources.
> > 
> > 
> > And were did you get this values from exactly?
> 
> I reviewed several websites; BG-wiki if they had any, then I branched out to other various sites like ffxiah, ffxiopedia, and various pictures on google image search of sell numbers. I tried jp-wiki but couldn't read/translate fast enough. Initially, this was to fix the sell base number of Girandola due to the report but I just saw a bunch of 0 sell base numbers and thought about due diligence. So, I just did the ones around that area of the sql. If this isn't doesn't comply, I can withdraw this no problem.

We use an NPC on retail unaffected by fame, wiki's have repeatedly proven to be a crapshoot on this info. There was an in-file comment directing people to a reliable NPC. While its simply not possible for one person to check every sellable item in the game we've been crowdsourcing it for awhile now.

**Comment by TeoTwawki** [comment]
looking at the items you are doing here, I think i actually have 2/3 of these and can look over the weekend.

**Comment by Aeltorian** [comment]
> 

My apologies. I've also learned that several of these may been intentionally set to 0 to prevent Gil farming. I was unaware as I didn't see any notation stating so. I will learn from this.

**Comment by zach2good** [comment]
The NPC in question:
https://github.com/LandSandBoat/server/blob/base/sql/item_basic.sql#L39-L45
```
/*
    #############################################################################
    # A Note regarding BaseSell values: When obtaining/updating Vendor buyback  #
    # prices, please be certain to use an NPC which ignores Fame as a factor    #
    # when buying items from the player. One such verified example is Challoux  #
    # in Port Jeuno (https://www.bg-wiki.com/ffxi/Challoux). This will ensure   #
    # that buyback prices are as accurate as possible.                          #
    #############################################################################
*/;

```

File: sql/item_mods.sql (hunk 1 above)
Comment (CatsEyeXI) [reply sql/item_mods.sql:67303]: > I thought I did. Carver was walking me through and we did make a branch of it.
> 
> edit: wait, why is this edit here too? Eh, i'm still learning this structuring of this. My apologies. I may have likely not a proper pull before starting the new branch therefore catching the other edit. Again, my apologies. Rough start. Lol

yeah, remember, last thing i told you last night is to make sure you switch back to your base each time, as this is what will happen if you dont :) 

I can help you fix this later on...

**Pull request closed without merging**